lib/trino-filesystem/src/main/java/io/trino/filesystem/tracing/Tracing.java

```diff
@@ -21,17 +21,17 @@
 
 import java.util.Optional;
 
-final class Tracing
+public final class Tracing
```

Comment:
this could be extracted to a separate commit

Reply:
it doesn't seem that useful to make methods public as a separate commit